packages/core/src/extensions/SideMenu/SideMenuPlugin.ts

```diff
@@ -348,6 +348,10 @@ export class SideMenuView<
 
     this.hoveredBlock = block.node;
 
+    if (!this.hoveredBlock?.hasAttribute("data-id")) {
```

Comment:
@matthewlipski can you review this, and also see if this check should be here or not? Could it also be that `hoveredBlock` shouldn't have been set in the first place if it doesn't have data-id?